broadcom-bnxt-en-8.3.cfg

```diff
@@ -0,0 +1,6 @@
+PACK_DESC="Driver Pack for Broadcom bnxt_en"
+PACK_UUID="6854ab19-f807-4444-b94f-3ce7d64efe50"
+PACK_BUILD="pack.0.1"
+RPM_FILE="broadcom-bnxt-en-1.10.3_232.0.155.5-1.xcpng8.3.x86_64.rpm"
+RPM_SHA256="3ee4ccaa267fb6e15b183ca1fb967a0859c9d5469a776370abbc3dc7592ed648"
+GPG_PUBKEY_FILE="RPM-GPG-KEY-XS-DDK-Test"
```

Comment:
Didn't we switch to `GPG_PUBKEY_FILE="RPM-GPG-KEY-XCP-SingleUse"`? See the previous update to a driver disk definition.